pom.xml

```diff
@@ -96,7 +96,7 @@
     <argLine>-Xms256m -Xmx512m</argLine>
 
     <!-- default bytecode version for src/main -->
-    <main.java.version>11</main.java.version>
```

Comment:
Hi @vidakovic I'm not comfortable at this stage on dropping java 11 support.

Any change on using an older version of `postgis` to avoid this?

Reply:
Well, premier support for JDK 11 already ended - I think - in September 2023 and extended support (I think this is only for paying customers) ends in 2 years from now? My argument would be that everything else (read: Spring/Spring Boot) is also moving forward and probably most of the users are not paying for extended JDK support. Just my 2 cents.

Reply:
I agree, maybe I can change on a 7.0 release... but I don't want to break compatibility on 6.x train

Reply:
Sounds reasonable to me. Thanks